cylc/flow/cfgspec/globalcfg.py

```diff
@@ -168,16 +168,69 @@
                 restarted (see:ref:`auto-stop-restart`).
             ''')
             Conf('ranking', VDR.V_STRING, desc='''
-                A multiline string containing Python expressions to filter
-                and/or rank hosts. For example:
+                Rank and filter run hosts based on system information.
 
-                .. code-block:: none
+                This can be used to provide load balancing to ensure no one run
+                host is overloaded and provide thresholds beyond which Cylc
+                will not attempt to start new schedulers on a host.
 
-                   cpu_percent() < 70
+                .. _psutil: https://psutil.readthedocs.io/en/latest/
+
+                This should be a multiline string containing Python expressions
+                to rank and/or filter hosts. All `psutil`_ attributes are
+                available for use in these expressions.
+
+                .. rubric:: Ranking
+
+                Rankings are expressions which return numerical values.
+                The host which returns the lowest value is chosen. Examples:
+
+                .. code-block:: python
+
+                   # rank hosts by cpu_percent
                    cpu_percent()
 
-                To filter by ``cpu_percent() < 70`` then to rank by
-                ``cpu_percent``.
+                   # rank hosts by 15min average of server load
+                   getloadavg()[2]
+
+                   # rank hosts by the number of cores
+                   # (divide by 1 because the lowest value is chosen)
+                   1 / cpu_count()
```

Comment:
```suggestion
                   # rank hosts by the number of cores
                   # (take reciprocal because the lowest value is chosen)
                   1 / cpu_count()
```

Reply:
Changed to `-1 * cpu_count` which should be a little clearer.